src/components/Settings/SaveSearchItem.vue

```diff
@@ -1,61 +1,111 @@
 <template>
-	<article>
-		<div
-			class="tw-flex tw-flex-col"
-		>
-			<span
-				v-if="savedSearch.name"
-				class="tw-text-medium tw-m-1 tw-p-1 tw-font-medium"
+	<kv-settings-card
+		v-if="savedSearch.name"
+		:title="savedSearch.name"
+	>
+		<template #content>
+			<p
+				class="tw-text-small tw-mb-2"
 			>
-				{{ savedSearch.name }}
-			</span>
+				Sorted by {{ savedSearch.loanSearchCriteria.sortBy }}
+			</p>
 			<span>
 				<kv-checkbox
 					v-model="showAlerts"
-					class="tw-text-small tw-m-1 tw-p-1"
+					class="tw-mb-2"
 				>
 					Email me when new loans become available
 				</kv-checkbox>
 			</span>
-			<span>
-				<kv-button
-					variant="secondary"
-					class="tw-text-small tw-m-1 tw-p-1"
-				>
-					Delete Search
-				</kv-button>
-				<kv-button
-					variant="secondary"
-					class="tw-text-small tw-m-1 tw-p-1"
-				>
-					View Loans
-				</kv-button>
-			</span>
-		</div>
-	</article>
+			<hr
+				class="tw-border-tertiary tw-mb-2"
+			>
+			<kv-button
+				:href="savedSearch.url"
+				class="tw-mr-2 tw-mb-2 md:tw-mb-0"
+			>
+				View Loans
+			</kv-button>
+			<kv-button
+				variant="secondary"
+				@click="deleteSavedSearch(savedSearch.id)"
+			>
+				Delete Search
+			</kv-button>
+		</template>
+	</kv-settings-card>
 </template>
 
 <script>
+import gql from 'graphql-tag';
+import KvSettingsCard from '@/components/Kv/KvSettingsCard';
 import KvButton from '~/@kiva/kv-components/vue/KvButton';
 import KvCheckbox from '~/@kiva/kv-components/vue/KvCheckbox';
 
 export default {
 	name: 'SaveSearchItem',
 	data() {
 		return {
-			showAlerts: false,
+			showAlerts: this.savedSearch?.isAlert,
+			deleteSearch: () => []
 		};
 	},
 	inject: ['apollo', 'cookieStore'],
 	components: {
+		KvSettingsCard,
 		KvButton,
-		KvCheckbox,
+		KvCheckbox
 	},
 	props: {
 		savedSearch: {
 			type: Object,
 			default: () => {},
 		}
-	}
+	},
+	methods: {
+		deleteSavedSearch(id) {
+			this.apollo.mutate({
+				mutation: gql`mutation deleteSearch($id: Int!) {
+				my {
+					deleteSavedSearch(id: $id) 
+				}
+			}`,
+				variables: {
+					id
+				},
+			}).then(result => {
+				const deleteSearchData = result?.data?.my?.deleteSavedSearch;
+				this.$emit('delete-saved-search', { id: deleteSearchData ? id : null });
+			});
+		},
+		emailAlert(isAlert) {
+			this.apollo.mutate({
+				mutation: gql`mutation updateSavedSearch($id: Int!, $savedSearch: SavedSearchUpdateInput!) {
+					my {
+						updateSavedSearch(id: $id, savedSearch: $savedSearch){
+							id
+							name
+							isAlert
+						}
+					}
+				}`,
+				variables: {
+					id: this.savedSearch?.id,
+					savedSearch: { isAlert }
+				}
+			}).then(result => {
+				const emailAlertData = result?.data?.my?.updateSaveSearch;
+				console.log(emailAlertData);
```

Comment:
Do we need to do anything with the result for this mutation or can this `.then` be removed?

Reply:
No, I don't believe we do at this time. Good suggestion!